components/UserTestimonies/ViewTestimony.tsx

```diff
@@ -65,6 +67,9 @@ export const TestimonyItem = ({ testimony }: { testimony: Testimony }) => {
 
   const { result: bill } = useBill(testimony.billId)
 
+  const TESTIMONY_CHAR_LIMIT = 442
```

Comment:
interesting number choice 👍

Reply:
I'll get on the button right away. As for the choice of number, I literally counted the number of characters in the example image that Sasha provided me. I'm happy to change it to 440 or 450 if that makes more sense.    ;)

Reply:
I fixed the scrolling issue and confirmed the color for the dropdown background with James.